operations/configuration_management/Management_Node_Image_Customization.md

```diff
@@ -0,0 +1,163 @@
+# Management Node Image Customization
+
+**NOTE:** Some of the documentation linked from this page mentions use of the Boot Orchestration Service (BOS). The use of BOS
+is only relevant for booting compute nodes and can be ignored when working with NCN images.
+
+This document describes the configuration of a Kubernetes NCN image. The same steps could be used to modify a Ceph NCN image.
+
+1. (`ncn-mw#`) Locate the NCN image to be modified.
+
+    This example assumes that the administrator wants to modify the Kubernetes image that is currently in use by NCNs. However, the steps are the same for any NCN SquashFS image.
+
+    If the image to be modified is the image currently booted on an NCN, the value for `ARTIFACT_VERSION` can be found by looking
+    at the boot parameters for the NCNs, or from `/proc/cmdline` on a booted NCN. The version has the form of `X.Y.Z`.
```

Comment:
A link to relevant documentation for looking at boot parameters?

Reply:
Added.